sdk/src/subnet_id.rs

```diff
@@ -54,7 +51,8 @@ impl SubnetID {
     /// hosted in the current network. The rest of the route is left
     /// as-is. We only have information to translate from f2 to f0 for the
     /// last subnet actor in the root.
-    pub fn f0_id(&self, rt: &impl Runtime) -> SubnetID {
+    #[cfg(feature = "fil_actor")]
+    pub fn f0_id(&self, rt: &impl fil_actors_runtime::runtime::Runtime) -> SubnetID {
```

Comment:
It seems that adding the feature here broke the actors compilation.

Reply:
Thanks, typo, fixing now..